### ISSUE TITLE
exec user process caused "no such file or directory"

### ISSUE THREAD

**Issue by carlbennett** [factoriotools/factorio-docker#268]
Latest version (28598a42a33530a1d687c1dbbcfcf742f605dc02) seems to cause me problems:
```
[root@factorio-weekly ~]# docker start factorio ; docker logs -f factorio
factorio
standard_init_linux.go:207: exec user process caused "no such file or directory"
standard_init_linux.go:207: exec user process caused "no such file or directory"
standard_init_linux.go:207: exec user process caused "no such file or directory"
standard_init_linux.go:207: exec user process caused "no such file or directory"
```
The message repeats over and over. I'm not certain what it's looking for. All I did was:
```
  188  docker stop factorio
  189  docker rm factorio
  190  docker pull factoriotools/factorio
  191  docker run -d -p 34197:34197/udp -v /opt/factorio:/factorio --name factorio --restart=always factoriotools/factorio
```
I had anticipated there was an update for 0.17.54:
* https://www.reddit.com/r/factorio/comments/c928b4/version_01754/

I was wrong.

**Comment by carlbennett** [comment]
Linux Kernel: 5.1.12-300.fc30.x86_64
Docker: docker-ce-18.09.6-3.fc29.x86_64

**Comment by carlbennett** [comment]
0.17.53 tag seems to be good still.

**Comment by carlbennett** [comment]
Fixed in #272
